sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleOverviewView.groovy

```diff
@@ -54,9 +67,12 @@ class SampleOverviewView extends VerticalLayout implements HasHotbar, HasTitle,
         ListDataProvider<Sample> dataProvider = ListDataProvider.ofCollection(samples)
         samplesGrid.addColumn(Sample::getCode).setCaption("Sample Code").setId("SampleCode")
         samplesGrid.addColumn(Sample::getName).setCaption("Sample Name").setId("SampleName")
-        samplesGrid.addColumn(Sample::getStatus).setCaption("Sample Status").setId("SampleStatus").setStyleGenerator({Sample sample -> determineColor(sample.status)})
+        samplesGrid.addColumn(Sample::getStatus).setCaption("Sample Status").setId("SampleStatus")
+                .setStyleGenerator({Sample sample -> determineColor(sample.status)})
```

Comment:
What changed here? 🤔

Reply:
the extremely long line was formatted so that it is more readable. No code change